app/scripts/lib/app/bundles.js

```diff
@@ -63,7 +63,7 @@ export default {
     {
       "name": "textlint",
       "key": "textlint",
-      "version": "6.0.1",
+      "version": "6.0.2",
```

Comment:
Yes, `bundles.js` is automatically generated from installed packages by `npm install`.

Never mind as it should always be the latest version.